dataFields/ABFieldTextFormulaCore.js

```diff
@@ -548,29 +556,29 @@ function getNumberToWords(number) {
 }
 
 function getDateDayOfWeekName(abFactory, date) {
-   // var localizeDT = moment(date);
+   // const localizeDT = moment(date);
    // localizeDT.locale(AD.lang.currentLanguage);
    // return localizeDT.format("dddd");
 
    return abFactory.toDateFormat(date, {
       format: "dddd",
-      localeCode: AD.lang.currentLanguage,
+      localeCode: abFactory.Config._config.user.languageCode,
```

Comment:
this should be `abFactory.Account.language()`

Better yet, rename `abFactory`  to `AB` to maintain consistency with the rest of our code.